site/tests/integration/docs.test.ts

```diff
@@ -105,12 +105,10 @@ test("Docs page should contain key elements and interactions should work", async
     page.locator("text=PreviousEmbedding Blocks >> a"),
   ).toHaveAttribute("href", "/docs/embedding-blocks");
 
-  await expect(
-    page.locator("text=NextCore Specification 0.2 >> a"),
-  ).toBeVisible();
+  await expect(page.locator("text=NextCore Specification >> a")).toBeVisible();
 
   await expect(
-    page.locator("text=NextCore Specification 0.2 >> a"),
+    page.locator("text=NextCore Specification >> a"),
```

Comment:
```suggestion
    page.locator("text=NextCore >> a"),
```

Follows https://github.com/blockprotocol/blockprotocol/pull/518/commits/b3ee31071fe47c55d5e3096bbf7937573bc15a6f